config.py

```diff
@@ -12,23 +12,19 @@ class Config:
     SQLALCHEMY_COMMIT_ON_TEARDOWN = True
     SITE_URL = dotenv.get('SITE_URL')
```

Comment:
I actually did not have it in config at first but later decided it was better to have there and retrieve it from the config whenever I need it rather than from the environment but I can see I forgot to change the rest of my code to reflect this. I will do that right away, thank you